modules/channel-vonage/src/backend/client.ts

```diff
@@ -305,7 +315,7 @@ export class VonageClient {
   // TODO: add support for WhatsApp Templates instead of using down rendering
   private async sendOptions(event: sdk.IO.Event, text: string, options: MessageOption[]) {
     if (options.length) {
-      text = `${text}\n\n${options.map(({ label }, idx) => `${idx + 1}. ${label}`).join('\n')}`
+      text = `${text}\n\n${options.map(({ label }, idx) => `*(${idx + 1})* ${label}`).join('\n')}`
```

Comment:
what are the * for ?

Reply:
They allow writing text in bold. Here the goal was to simply change options display from :

1. test
2. test

To

**(1)** test
**(2)** test